lib/matplotlib/colors.py

```diff
@@ -614,25 +614,47 @@ def __copy__(self):
         cmapobject._global = False
         return cmapobject
 
+    def get_bad(self):
+        """Get the color for masked values."""
+        if not self._isinit:
+            self._init()
+        return self._lut[self._i_bad]
+
     def set_bad(self, color='k', alpha=None):
         """Set the color for masked values."""
         _warn_if_global_cmap_modified(self)
         self._rgba_bad = to_rgba(color, alpha)
         if self._isinit:
             self._set_extremes()
 
+    def get_under(self):
+        """
+        Get the color for low out-of-range values.
+        """
```

Comment:
```suggestion
        """Get the color for low out-of-range values."""
```
We squeeze single line docstrings on one line if it fits including the qoutes.